docs/api_reference.md

```diff
@@ -0,0 +1,6 @@
+# cluster-registry API Reference
```

Comment:
Is this file generated? It seems like it could be generated by the script that creates the API documentation.

Reply:
It is not generated. It could be, but it should only ever change whenever there are new API versions. I don't really have a strong opinion here. Let me know if you'd prefer it to be generated and I can do a follow-up PR.

Reply:
Thanks! Given the infrequency of changes, I think it's probably not worth generating this right now.